components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1246,6 +1241,9 @@ where
             }
             CasualMessage::SnapshotApplied => {
```

Comment:
Could you make it a `SignificantMsg`? Because `CasualMessage` may be dropped if the peer is busy.

Reply:
@overvenus It would make the code quite ugly if we change CasualMessage to SignificantMsg. This is because the Runner in Region.rs now needs to have two router instances, one for CasualMessage and the other for SignificantMsg. We cannot use one instance for both as in test code we use SyncSender for the parameter `router`, one instance simply cannot be both CasualRouter and SignificantRouter at the same time. 
On the other hand, I think it's acceptable to have this message dropped in very busy case because it's just to accelerate the destroy peer speed, and without it, the correctness is still not impacted.